bpfman/src/lib.rs

```diff
@@ -481,9 +481,10 @@ async fn setup() -> Result<(Config, Db), BpfmanError> {
 
     Ok((
         open_config_file(),
-        init_database(get_db_config())
-            .await
-            .expect("Unable to open root database"),
+        init_database(get_db_config()).await.unwrap_or_else(|err| {
+            eprintln!("{}", err);
+            std::process::exit(1)
+        }),
     ))
```

Comment:
I suggest we return an error here instead of just exiting.  The following should work:

```suggestion
    Ok((open_config_file(), init_database(get_db_config()).await?))
```